spec/requests/have_you_been_evicted_spec.rb

```diff
@@ -0,0 +1,52 @@
+RSpec.describe "have-you-been-evicted" do
+  describe "GET /have-you-been-evicted" do
+    let(:selected_option) { I18n.t("coronavirus_form.questions.have_you_been_evicted.options").sample }
+
+    context "without session data" do
+      it "shows the form" do
+        visit have_you_been_evicted_path
+
+        expect(page.body).to have_content(I18n.t("coronavirus_form.questions.have_you_been_evicted.title"))
+        I18n.t("coronavirus_form.questions.have_you_been_evicted.options").each do |option|
+          expect(page.body).to have_content(option)
+        end
+      end
+    end
+
+    context "with session data" do
+      before do
+        page.set_rack_session(have_you_been_evicted: selected_option)
+      end
+
+      it "shows the form with prefilled response" do
+        visit have_you_been_evicted_path
+
+        expect(page.body).to have_content(I18n.t("coronavirus_form.questions.have_you_been_evicted.title"))
+        expect(page.find("input#option_#{selected_option.parameterize.underscore}")).to be_checked
+      end
+    end
+  end
+
+  describe "POST /still-working" do
```

Comment:
```suggestion
  describe "POST /have-you-been-evicted" do
```

Reply:
ah don't know how I missed this, sorry, fixed